spec/javascripts/components/listing/listingContainer_spec.coffee

```diff
@@ -109,22 +109,69 @@ do ->
           expect(ctrl.lotteryResultsListings).toBeDefined()
 
       describe '$ctrl.isOwnershipListing', ->
+        testListing = angular.copy(fakeListing)
```

Comment:
very open to feedback on all of these tests!

I wasn't clear at all about when I need to angular.copy(fakeListing) and when I can just use it. Does it get reset every test?

Reply:
it doesn't get reset, so ideally we'd copy it for a test in which we are going to modify it. the existing tests haven't always been consistent about that, since in many cases tests add things to the fakeListing which don't affect other tests, but it will be more consistent for us to always copy so that we're always clear about what the listing being used in the test will contain